discordbot/OverworldDiscordBot.py

```diff
@@ -0,0 +1,282 @@
+# Needs a discord bot account and a server with all the mark ids as emojis.
```

Comment:
as it is a class in the reader now, comments like this should be moved to the readme along with a full section on the discord bot, section should include/link to a guide on how to set up a bot and obtain its token

Reply:
Note for future: command/option to have the bot automatically add mark emojis to your discord server, and to have them stored for use with the gui instead of pulling from serebii

Reply:
Added a tiny section to the readme, removed the comment, and added new items to the future improvements list.